beanstalk_worker/worker_start_test.py

```diff
@@ -208,7 +208,17 @@ def test_job_data(job_data):
     try:
         job = bs.reserve()
         job_data = json.loads(job.body)
-        test_job_data(job_data)
+        if len(sys.argv) > 1 and sys.argv[1] == "ci":
+            # If working in CI environment, don't run the scanner
+            logger.log(level=logging.INFO, msg="CI check...")
+            job_data["action"] = "start_delivery"
+            job_data["msg"] = ""
+            job_data["logs"] = ""
+            bs.use("master_tube")
+            jid = bs.put(json.dumps(job_data))
+        else:
+            test_job_data(job_data)
+
```

Comment:
The code changes proposed in this PR is already part of master branch check https://github.com/CentOS/container-pipeline-service/blob/master/beanstalk_worker/worker_start_test.py#L211-L221 

and the purpose of the PR is achieved, i.e. not to run the scanner if executing in `ci` env.